codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/protocol/ServerProtocolTestGenerator.kt

```diff
@@ -286,23 +298,20 @@ class ServerProtocolTestGenerator(
         val operationName = "${operationSymbol.name}${ServerHttpProtocolGenerator.OPERATION_INPUT_WRAPPER_SUFFIX}"
         rustWriter.rustTemplate(
             """
-            let http_request = http::Request::builder()
-                .uri(${testCase.uri.dq()})
-                .header("Content-Type", ${testCase.bodyMediaType.orNull()?.dq()})
-                .body(#{SmithyHttpServer}::Body::from(#{Bytes}::from_static(b${body.dq()})))
-                .unwrap();
             use #{Axum}::extract::FromRequest;
             let mut http_request = #{Axum}::extract::RequestParts::new(http_request);
             let input_wrapper = super::$operationName::from_request(&mut http_request).await.expect("failed to parse request");
             let input = input_wrapper.0;
             """,
             *codegenScope,
         )
-        if (body == "") {
-            rustWriter.write("// No body")
-            rustWriter.write("assert_eq!(std::str::from_utf8(input).expect(\"`body` does not contain valid UTF-8\"), ${"".dq()});")
+        if (operationShape.outputShape(model).hasStreamingMember(model)) {
+            rustWriter.rust(
+                """// TODO: add support for streaming types
+            assert!(false);"""
```

Comment:
I'd suggest `panic!("streaming types aren't supported")` to make the failure a little easier to diagnose

Reply:
Yep, good idea!